src/appkit.cs

```diff
@@ -23721,4 +24387,592 @@ interface NSMenuValidation
+
+		[Static]
+		[Export ("strokeColorPickerWithIdentifier:")]
+		NSColorPickerTouchBarItem CreateStrokeColorPicker(string identifier);
+
+		[Static]
+		[Export ("colorPickerWithIdentifier:buttonImage:")]
+		NSColorPickerTouchBarItem CreateColorPicker (string identifier, NSImage image);
+
+		[Export ("color", ArgumentSemantic.Copy)]
+		NSColor Color { get; set; }
+
+		[Export ("showsAlpha")]
+		bool ShowsAlpha { get; set; }
+
+		[Export ("colorList", ArgumentSemantic.Strong)]
+		NSColorList ColorList { get; set; }
+
+		[Export ("customizationLabel")]
+		string CustomizationLabel { get; set; }
+
+		[NullAllowed, Export ("target", ArgumentSemantic.Weak)]
+		NSObject Target { get; set; }
+
+		[NullAllowed, Export ("action", ArgumentSemantic.Assign)]
+		Selector Action { get; set; }
+
+		[Export ("enabled")]
+		bool Enabled { [Bind ("isEnabled")] get; set; }
+	}
+
+	[Mac (10,12,1)]
+	[BaseType (typeof(NSTouchBarItem))]
+	interface NSCustomTouchBarItem
+	{
+		[Export ("initWithIdentifier:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string identifier);
+
+		[Export ("view", ArgumentSemantic.Strong)]
+		NSView View { get; set; }
+
+		[Export ("viewController", ArgumentSemantic.Strong)]
+		NSViewController ViewController { get; set; }
+
+		[Export ("customizationLabel")]
+		string CustomizationLabel { get; set; }
+	}
+
+	[Category]
+	[BaseType (typeof(NSGestureRecognizer))]
+	interface NSGestureRecognizer_NSTouchBar
+	{
+		[Mac (10, 12, 1)]
+		[Export ("allowedTouchTypes", ArgumentSemantic.Assign)]
+		NSTouchTypeMask GetAllowedTouchTypes ();
+
+		[Mac (10, 12, 1)]
+		[Export ("setAllowedTouchTypes:", ArgumentSemantic.Assign)]
+		void SetAllowedTouchTypes (NSTouchTypeMask types);
+	}
+
+	[Mac (10,12,1)]
+	[BaseType (typeof(NSTouchBarItem))]
+	interface NSGroupTouchBarItem
+	{
+		[Export ("initWithIdentifier:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string identifier);
+
+		[Static]
+		[Export ("groupItemWithIdentifier:items:")]
+		NSGroupTouchBarItem CreateGroupItem (string identifier, NSTouchBarItem[] items);
+
+		[Export ("groupTouchBar", ArgumentSemantic.Strong)]
+		NSTouchBar GroupTouchBar { get; set; }
+
+		[Export ("customizationLabel")]
+		string CustomizationLabel { get; set; }
+	}
+
+	[Mac (10,12,1)]
+	[BaseType (typeof(NSTouchBarItem))]
+	interface NSPopoverTouchBarItem
+	{
+		[Export ("initWithIdentifier:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string identifier);
+
+		[Export ("popoverTouchBar", ArgumentSemantic.Strong)]
+		NSTouchBar PopoverTouchBar { get; set; }
+
+		[Export ("customizationLabel")]
+		string CustomizationLabel { get; set; }
+
+		[Export ("collapsedRepresentation", ArgumentSemantic.Strong)]
+		NSView CollapsedRepresentation { get; set; }
+
+		[NullAllowed, Export ("collapsedRepresentationImage", ArgumentSemantic.Strong)]
+		NSImage CollapsedRepresentationImage { get; set; }
+
+		[Export ("collapsedRepresentationLabel", ArgumentSemantic.Strong)]
```

Comment:
/11x/AppKit.framework/Headers/NSPopoverTouchBarItem.h:@property (strong) NSString *collapsedRepresentationLabel;

I'm not making it up.